qurator/dinglehopper/ocr_files.py

```diff
@@ -104,6 +91,35 @@ def page_extract(tree, *, textequiv_level="region"):
     return ExtractedText(None, regions, "\n", None)
 
 
+def extract_texts_from_reading_order_group(group, tree, nsmap, textequiv_level):
```

Comment:
There is a method [`get_AllRegions`](https://github.com/OCR-D/core/blob/master/ocrd_models/ocrd_page_user_methods/get_AllRegions.py) in core that would probably be handy here. It can list all TextRegions in document-order, reading-order+document-order and reading-order only.

Reply:
dinglehopper does not use OCR-D core for anything other than the OCR-D CLI (yet?)

Reply:
I'm aware of that, just saying that this particular problem was solved in core.

Reply:
Imho: switching from the "custom" implementation to core (althouth beneficial) should be a conscious decision and handled separately.